test/jdk/javax/net/ssl/SSLEngine/Basics.java

```diff
@@ -26,34 +26,47 @@
  * @bug 4495742
  * @summary Add non-blocking SSL/TLS functionality, usable with any
  *      I/O abstraction
- *
  * This is intended to test many of the basic API calls to the SSLEngine
  * interface.  This doesn't really exercise much of the SSL code.
  *
+ * @library /test/lib
  * @author Brad Wetmore
+ * @run main/othervm Basics
  */
 
 import java.security.*;
 import java.io.*;
 import java.nio.*;
+import java.util.Arrays;
 import javax.net.ssl.*;
 import javax.net.ssl.SSLEngineResult.*;
 
+import jdk.test.lib.security.SecurityUtils;
+
 public class Basics {
 
-    private static String pathToStores = "../etc";
-    private static String keyStoreFile = "keystore";
-    private static String trustStoreFile = "truststore";
-    private static String passwd = "passphrase";
+    private static final String pathToStores = "../etc";
+    private static final String keyStoreFile = "keystore";
+    private static final String trustStoreFile = "truststore";
+    private static final String passwd = "passphrase";
```

Comment:
passwd variable defined here is unused. Also can you please capitalize all the constants.